server-common/src/main/java/com/datastrato/gravitino/server/web/CorsFilterHelper.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.server.web;
+
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.eclipse.jetty.servlets.CrossOriginFilter;
+
+class CorsFilterHelper {
+
+  static final String PREFLIGHT_MAX_AGE = "preflightMaxAge";
+
+  private CorsFilterHelper() {}
+
+  public static FilterHolder createCorsFilterHolder(JettyServerConfig config) {
```

Comment:
And you can simplify the method name to `create` to remove duplication with class name.

Reply:
Fixed.